### PR TITLE
Added input image input

### PR DESCRIPTION
**Pull request by bentocg**
As soon as rasterio gets updated for virtual machines on bridges, this should fix the output from @iparask 's EnTK script.

### CODE REVIEW

**Review by bspitzbart — COMMENTED** [review]
image input change looks good to me.  
But .idea .Rhistory etc. reappeared (probably from the merge with master) Can I remove them or do we need them?

**Comment by bspitzbart** [comment]
Just noticed this is a request to merge into master? I will change it to devel.

**Comment by bentocg** [comment]
Right, I meant devel. We should get rid of those, I should have added one by one instead of using git add . But still, why aren't those in the .gitignore???

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,7 @@ __pycache__
 .ipynb_checkpoints/
 "model_search_stable.ipynb" 
 "model_search_experimental.ipynb" 
+.gitignore
+.pylintrc
```
Comment (iparask) [inline .gitignore]: This is definitely needed, otherwise Travis cannot execute the linting process.

**Comment by iparask** [comment]
@bspitzbart I think we can merge this if it is okay with you as well.

**Comment by bspitzbart** [comment]
Have you run the 5 test images so we can verify the output?

**Comment by iparask** [comment]
Yes, I have. The images are in the VM. I am trying to follow the instructions in #17 to validate, but still I am missing information. The images are `/home/iparask/ICEBERG/testingSeals/src/entk_script/validation`.

**Comment by bspitzbart** [comment]
The output looks good. Except the first image (WV03_20170202190457) is missing location files, but has predictions.
The other image locations look good.  The scene column is now correct. The x,y are still image coordinates, but project on the image correctly in arcGIS.
@bentocg can you verify that is what is expected?

**Review by bspitzbart — APPROVED** [review]

**Comment by bentocg** [comment]
> The output looks good. Except the first image (WV03_20170202190457) is missing location files, but has predictions.
> The other image locations look good. The scene column is now correct. The x,y are still image coordinates, but project on the image correctly in arcGIS.
> @bentocg can you verify that is what is expected?

That is totally fine, that scene had no location files because there weren't any predicted seals for that scene (which is actually expected since there aren't any seals in there)

**Pull request merged**